pytorch_lightning/trainer/training_loop.py

```diff
@@ -876,11 +877,22 @@ def build_train_args(self, batch, batch_idx, opt_idx, hiddens):
                 )
 
         # pass hiddens if using tbptt
-        if self.trainer.truncated_bptt_steps is not None:
+        if self._truncated_bptt_enabled():
             args.append(hiddens)
 
         return args
 
+    def _truncated_bptt_enabled(self) -> bool:
+        """ Temporary tbptt utilities until this flag is fully migrated to the lightning module. """
+        return self._truncated_bptt_steps() > 0
+
+    def _truncated_bptt_steps(self) -> int:
```

Comment:
Any reason why it is not a property ?

Reply:
I didn't want to expose it to callers just in case someone's doing `trainer.train_loop.truncated_bptt_steps`

Reply:
can it be protected property?